forte/data/ontology/core.py

```diff
@@ -429,7 +432,20 @@ def _resolve_pointer(self, ptr: BasePointer) -> Entry:
         if isinstance(ptr, Pointer):
             return self.pack.get_entry(ptr.tid)
         elif isinstance(ptr, MpPointer):
-            return self.pack.packs[ptr.pack_index].get_entry(ptr.tid)
+            # bugfix/new feature 559: in new version pack_index will be using pack_id internally
+            pack_array_index = ptr.pack_index  # old version
+            pack_version = [0, 0, 0]
+            try:
+                pack_version = self.pack.pack_version
+            except AttributeError as ae:
+                print(ae)  # the version is not set : print or log the exception
```

Comment:
Is this an unhandled exception, shall we simply throw the exception, i.e. terminate with exceptions instead of simply `print`? Also `print` is not the recommended way to log, we should use the logging module at the least